src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160715_execution_key_default.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="execution_key_default">
+        <sql>
+            update WORKFLOW set EXEC_SERVICE_KEY = 'default' where EXEC_SERVICE_KEY is null and EXTERNAL_ID is not null;
```

Comment:
I would just set it to cromwell1 since that is what it really is. I don't think we should have any notion of default.